packages/cli-kit/src/public/node/session.ts

```diff
@@ -62,25 +62,35 @@ ${outputToken.json(scopes)}
 /**
  * Ensure that we have a valid session to access the App Management API.
  *
- * @param scopes - Optional array of extra scopes to authenticate with.
+ * @param appManagementScopes - Optional array of extra scopes to authenticate with.
+ * @param businessPlatformScopes - Optional array of extra scopes to authenticate with.
  * @param env - Optional environment variables to use.
  * @param options - Optional extra options to use.
  * @returns The access token for the App Management API.
  */
-export async function ensureAuthenticatedAppManagement(
-  scopes: AppManagementAPIScope[] = [],
+export async function ensureAuthenticatedAppManagementAndBusinessPlatform(
```

Comment:
I find it hard to imagine this being the case, but lets be sure there's no consumers of `ensureAuthenticatedAppManagement` that are going to break.